scripts/manualAuditDataRequests/sendResults/sendSQSMessageToCompletedQueue.ts

```diff
@@ -1,10 +1,10 @@
 import { sendSqsMessage } from '../../../src/sharedServices/queue/sendSqsMessage'
 import { sendManualQueryPayload } from './sendManualQueryPayload'
-import { getEnv } from '../../../src/utils/helpers'
 
 export const sendSQSMessageToCompletedQueue = async (
+  environment: string,
   manualQueryCompleteSqsPayload: sendManualQueryPayload
```

Comment:
While we're here, could we make the `sendManualQueryPayload` match the standard uppercase for types rule?

Reply:
Fixed! Move it into the types folder too